Release 328.0.0 (#5482)
MetaMask/core: packages/assets-controllers/CHANGELOG.md

## [Unreleased]

## [53.1.1]

### Fixed

- Check if `KeyringController` is unlocked before processing account events in `MultichainBalancesController` ([#5473](https://github.com/MetaMask/core/pull/5473))

>   - This is needed since some Snaps might encrypt their state which needs the `KeyringController` to be unlocked.

Reviewer: ```suggestion
  - This is needed since some Snaps might decrypt their state which needs the `KeyringController` to be unlocked.
```

## [Unreleased]

## [53.2.0]

> ### Changed

Reviewer: This is probably only "fixes" no? So we could change it to a "Fixed" instead? (and we should change the version to be 53.1.1 probably?)

> - Adds safeguard for keyring being locked or unlocked ([#5473](https://github.com/MetaMask/core/pull/5473))

Reviewer: ```suggestion
- Check if `KeyringController` is unlocked before processing account events in `MultichainBalancesController` ([#5473](https://github.com/MetaMask/core/pull/5473))
  - This is needed since some Snaps might encrypt their state which needs the `KeyringController` to be unlocked.
```

> - Release 327.0.0 ([#5481](https://github.com/MetaMask/core/pull/5481))

Reviewer: This was bumping a dev deps, we can omit this one.

```suggestion
```

> - Publish example controllers and service as @metamask/sample-controllers ([#5363](https://github.com/MetaMask/core/pull/5363))

Reviewer: This was just fixing a comment on some test files, we can also omit.

```suggestion
```

> - Fix case when metadata are null ([#5455](https://github.com/MetaMask/core/pull/5455))

Reviewer: Looks like a fixed, not a changed:

```suggestion

## Fixed

- Fix runtime error in NFT detection when metadata is `null` ([#5455](https://github.com/MetaMask/core/pull/5455))
```

- Use Ethers for AssetsContractController ([#845](https://github.com/MetaMask/core/pull/845))

> [Unreleased]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@53.2.0...HEAD
> [53.2.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@53.1.0...@metamask/assets-controllers@53.2.0

Reviewer: ```suggestion
[Unreleased]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@53.1.1...HEAD
[53.2.0]: https://github.com/MetaMask/core/compare/@metamask/assets-controllers@53.1.0...@metamask/assets-controllers@53.1.1
```
